src/main/scala/com/tapad/docker/ComposeFile.scala

```diff
@@ -247,6 +261,38 @@ trait ComposeFile extends SettingsHelper with ComposeCustomTagHelpers with Print
     }
   }
 
+  def getStaticPortMappings(
+    serviceName: String,
+    ports: Seq[String],
+    usedStaticPorts: scala.collection.mutable.Set[String]
+  ): java.util.ArrayList[String] = {
+    val dynamicPortIdentifier = "0"
+
+    def checkUsed(portMapping: String): String = {
+      if (usedStaticPorts add portMapping)
```

Comment:
We should be consistent in our calling convention: `usedStaticPorts.add(portMapping)`